### PR TITLE
use minimal resource requests

### PR DESCRIPTION
**Pull request by phoracek**
Save some resources on our limited OKD clusters.

Signed-off-by: Petr Horacek <phoracek@redhat.com>

### CODE REVIEW

**Comment by phoracek** [comment]
ci test please

**Comment by phoracek** [comment]
ci test please

**Comment by phoracek** [comment]
ci test please

**Comment by phoracek** [comment]
ci test please

**Comment by phoracek** [comment]
ci test please

**Comment by phoracek** [comment]
It fails on OKD. I have a suspicion it is due to insufficient resources on the kubevirtci cluster. Will add more debugging information.

File: data/nmstate/003-operator.yaml [hunk 1]
```diff
@@ -31,6 +31,13 @@ spec:
           - --v=production
           image:  {{ .NMStateHandlerImage }}
           imagePullPolicy: {{ .ImagePullPolicy }}
+          resources:
+            requests:
+              cpu: "100m"
```
Comment (qinqon) [inline data/nmstate/003-operator.yaml]: 60m ?

File: data/ovs/002-ovs-daemonset.yaml [hunk 2]
```diff
@@ -44,6 +44,13 @@ spec:
         - name: ovs-cni-marker
           image: {{ .OvsMarkerImage }}
           imagePullPolicy: {{ .ImagePullPolicy }}
+          resources:
+            requests:
+              cpu: "100m"
```
Comment (qinqon) [inline data/ovs/002-ovs-daemonset.yaml]: 60m ?

File: data/nmstate/003-operator.yaml (hunk 1 above)
Comment (phoracek) [reply data/nmstate/003-operator.yaml]: Calling `nmstatectl` Python application is expensive. I'm afraid even 60m will not be enough. Let's keep it on 100 for now.

File: data/ovs/002-ovs-daemonset.yaml (hunk 2 above)
Comment (phoracek) [reply data/ovs/002-ovs-daemonset.yaml]: Did not work with 60m really, plus when I tried it manually, it was too slow.

**Pull request merged**